mobile/src/screens/meditation-list/components/timer-modal/libs/constants.ts

```diff
@@ -0,0 +1,19 @@
+import { SECONDS_IN_MINUTE } from '#libs/helpers/helpers';
+
+const DEFAULT_DURATION = 0;
```

Comment:
Let's move to shared as we are using on both mobile and frontend

Reply:
it's still here

Reply:
Because we don't have such a constant on the frontend